messages/en.json

```diff
@@ -430,6 +430,7 @@
     "werekitten_notify": "You are a \u0002werekitten\u0002. Due to your overwhelming cuteness, the seer always sees you as villager and the gunner will always miss you. Detectives can still reveal your true identity, however. Use \"kill <nick>\" to kill a villager.",
     "warlock_notify": "You are a \u0002{0}warlock\u0002. Each night you can curse someone with \"curse <nick>\" to turn them into a cursed villager, so the seer sees them as wolf. Act quickly, as your curse applies as soon as you cast it! Only detectives can reveal your true identity, seers will see you as a regular villager.",
     "wolf_mystic_notify": "You are a \u0002wolf mystic\u0002. Each night you divine the number of alive good villagers who have a special role. You may also use \"kill <nick>\" to kill a villager.",
+    "wolf_shaman_notify": "You are a \u0002wolf shaman\u0002. Each night, you may give a totem to someone using \"give <nick>\" in PM. You may also use \"kill <nick>\" to kill a villager. You must give a totem every night, and you may kill as well. If you don't give your totem, a random target will receive it.",
```

Comment:
Should mimic shaman_notify as much as possible for consistency:
You are a \u0002wolf shaman\u0002. You can select a player to receive a totem each night by using \"give <nick>\". You may give yourself a totem, but you may not give the same player a totem two nights in a row. If you do not give the totem to anyone, it will be given to a random player. You may also use \"kill <nick>\" to kill a villager.